frontend/src/css/modules/teams.scss

```diff
@@ -150,11 +150,11 @@ span.link-team-user {
 }
 
 .team-heading {
-    margin-top: -5px;
-    margin-bottom: 30px;
+    margin: 30px;
```

Comment:
I think simply removing `margin-top` will solve the issue. can you please try that.

Reply:
I did the same at first but it was not fully responsive. Example: For a screen resolution 768x642, it gave the following result.

![untitled](https://user-images.githubusercontent.com/22499864/31305249-e001e916-ab52-11e7-9d15-4126a71bf49b.jpg)

Reply:
Ahh..I see.

Reply:
@geekayush please use media query to achieve this responsive result. cc @RishabhJain2018